service/rttests/rt_test.go

```diff
@@ -71,12 +82,77 @@ var failureAttributeSets = [][]string{
 		"https://example.com/attr/cards/value/king"},
 }
 
+var successTokenExchangeAttributeSets = [][]string{
+	{"https://example.com/attr/language/value/english"},
+	{"https://example.com/attr/color/value/red", "https://example.com/attr/color/value/blue"},
+	{"https://example.com/attr/color/value/red", "https://example.com/attr/color/value/blue", "https://example.com/attr/color/value/green"},
+	{"https://example.com/attr/color/value/red", "https://example.com/attr/color/value/blue"},
+	{"https://example.com/attr/color/value/red", "https://example.com/attr/color/value/blue", "https://example.com/attr/color/value/green"},
+	{"https://example.com/attr/cards/value/jack"},
+	{"https://example.com/attr/language/value/english",
+		"https://example.com/attr/color/value/red",
+		"https://example.com/attr/color/value/blue",
+		"https://example.com/attr/cards/value/jack"},
+	{"https://example.com/attr/language/value/english",
+		"https://example.com/attr/color/value/red",
+		"https://example.com/attr/color/value/blue",
+		"https://example.com/attr/color/value/green",
+		"https://example.com/attr/cards/value/jack"},
+}
+
+var failureTokenExchangeSets = [][]string{
+	{"https://example.com/attr/language/value/french"},
+	{"https://example.com/attr/language/value/english", "https://example.com/attr/language/value/french"},
+	{"https://example.com/attr/color/value/red"},
```

Comment:
possible bug: what is the expected behavior for anyOf with multiple entities in the chain?

Reply:
@ttschampel  all entities need anyOf?

Reply:
^ ok that is how its functioning now, i vaguely remember with v1 we had it be permissive so if any entity fufilled anyOf then it passed